pagopa/io-app
# feat(Sicilia Vola): [IASV-13] Add SiciliaVola entry point

## Short description
This PR add the possibility to use the CTAs in the service details as entry points.

## List of changes proposed in this pull request
- Added the voucher list screen stub
- Added the voucher generation and the voucher list in the list of the route allowed to be used in the external CTA


https://user-images.githubusercontent.com/11773070/126797573-f5078fb5-3ed7-477f-b304-6dd043468615.mp4


## How to test
Make sure to checkout this branch `IASV-13-add-sicilia-vola-service` or to pull the last commit in the master if the [PR](https://github.com/pagopa/io-dev-api-server/pull/56) is already merged
